PhysicsTools/Heppy/python/analyzers/objects/TauAnalyzer.py

```diff
@@ -92,6 +91,7 @@ def id6(tau,X):
             elif tau.tauID(self.cfg_ana.tauLooseID):
                 event.looseTaus.append(tau)
                 event.inclusiveTaus.append(tau)
+        event.looseTaus.extend(event.selectedTaus)
 
```

Comment:
@gpetruc @cbernet , hi guys this makes sense to me... why should the loose being exclusive with the selected? if you agree we will make the same PR against colin branch